src/deephaven/plot/express/data_generator/example_data.py

```diff
@@ -0,0 +1,170 @@
+from __future__ import annotations
+
+from deephaven.replay import TableReplayer
+from deephaven.time import parse_instant
+from deephaven.table import Table
+from deephaven import empty_table
+from deephaven.time import (
+    parse_instant,
+    plus_period,
+    MINUTE,
+    seconds_to_nanos,
+)
+from deephaven.updateby import rolling_sum_tick, ema_tick
+import random, math
+
+
+def example_data(ticking: bool = True, hours_of_data: int = 1) -> Table:
+    """Returns a Deephaven table containing a generated example data set.
+    Data is 1 hour of randomly generated (but deterministic) fictional
+    stock market data, and starts with the first 5 minutes of data
+    already initilized so your example plots won't start empty.
+
+    Contains the following columns:
+    - timestamp: a starting from the date deephaven.io was registered
+    - sym: a string represent a fictional stock symbol
+    - exchange: a string representing a fictional stock exchange
+    - size: the number of shares in the trade
+    - price: the transaction price of the trade
+    - dollars: the dollar value of the trade (price * size)
+    - side: buy or sell side of the trade
+    - SPet500: A comparison to a fictional index
+    - index: an incrementing row index
+    - random: A random gaussian value using row index as seed
+
+    Args:
+      ticking: bool:  (Default value = True)
+        If true, the table will tick using a replayer, if
+        false the whole table will be returned as a static table.
+      hours_of_data: int: (Default value = 1)
+        The number of hours of data to return
+
+    Returns:
+      A Deephaven Table
+
+    """
+
+    base_time = parse_instant(
+        "2018-06-01T08:00:00 ET"
+    )  # day deephaven.io was registered
+    sym_list = ["CAT", "DOG", "FISH", "BIRD", "LIZARD"]
+    sym_weights = [95, 100, 70, 45, 35]
+    exchange = ["NYPE", "PETX", "TPET"]
+    exchange_weights = [50, 100, 45]
+    init_value = 100  # is used inside query string
+
+    ticks_per_second = 10
+    size_in_seconds: int = hours_of_data * 60 * 60 * ticks_per_second
+
+    # base time is 8am, start at 9am so there's already data showing and tick until 5pm
+    start_time = plus_period(base_time, (int)(MINUTE * 5))
+    end_time = plus_period(base_time, seconds_to_nanos(size_in_seconds))
+
+    def random_gauss(seed: int) -> float:
+        random.seed(seed)  # set seed once per row
+        return random.gauss(0, 1)
+
+    def random_list_sym(seed: int) -> str:
+        return random.choices(sym_list, sym_weights)[0]
+
+    def list_sym_index(sym: str) -> int:
+        return sym_list.index(sym)
```

Comment:
This is going to be doing a linear time lookup on sym_list every time the function is called, which although is unlikely to be a huge problem due to the short list, isn't necessary. 
There are a few ways to do this, but I'm thinking something like `sym_dict = { sym: i for i, v in enumerate(sym_list) }` below line 50 then line 71 becomes
`return sym_dict[sym]`

Reply:
good suggestion, I can skip the function too and just do that in the query string